mlflow/utils/autologging_utils/__init__.py

```diff
@@ -55,6 +58,11 @@
 # Exempts autologging flavors from the `_AUTOLOGGING_GLOBALLY_DISABLED` flag
 _AUTOLOGGING_GLOBALLY_DISABLED_EXEMPTIONS = []
 
+# Flag indicating whether autologging is thread-locally disabled for all integrations.
+_AUTOLOGGING_THREAD_LOCALLY_DISABLED = ThreadLocalVariable(default_factory=lambda: False)
```

Comment:
Having multiple flags for autolog disablement looks confusing. Can we use a single flag?

Reply:
if we want to support both "thread locally disablement" and "globally disablement",  we have to use 2 flags, one for the global, another for thread local, can't merge them.